packages/itwin/grouping-mapping-widget/src/components/Groups/GroupListItem.tsx

```diff
@@ -24,45 +25,44 @@ export interface GroupItemProps extends Omit<GroupsProps, "onClickAddGroup"> {
   ) => void;
 }
 
-export const GroupItem = ({
+export const GroupListItem = ({
```

Comment:
Is there an easy way to check what is exported from the package? I see you are renaming components, so I am not that confident that this will not cause breaking changes.

Reply:
@Guillar1 Could you set it up in a separate PR?